BioArchLinux/snpeff/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer: Bipin Kumar <bipin@ccmb.res.in>
+
+pkgname=snpeff
+pkgver=5.1
+pkgrel=1
+pkgdesc="Genetic variant annotation and effect prediction toolbox. https://doi.org/10.4161/fly.19695"
+arch=('any')
+url="https://pcingola.github.io/SnpEff/"
+provides=(""${pkgname%-pkgver}"")
+license=('MIT')
+depends=('java-runtime>=12''python''perl''bash')
```

Comment:
did you forget to add spaces between dependencies?

Reply:
i was advised to get SnpEff package from sources. maybe it would be better to remove the pull request for this and try later?